source/_docs/automation/troubleshooting.markdown

```diff
@@ -3,41 +3,68 @@ title: "Troubleshooting automations"
+
+## Traces
+
 When an {% term automation %} is run, all steps are recorded and a trace is made. From the UI, open **Settings**, which is located in the sidebar, then select **Automations & Scenes** to go to the automation editor or click this button directly: {% my automations badge %}
 
-Click on the clock icon next to an automation to enter the debugging screen. Alternatively, click on **Show trace** directly from a Logbook automation entry.
+From the automation editor UI, or in the automations list in the 3-dots menu, click **Traces**. Alternatively, click on an automation entry shown in the Logbook.
 
 ![Automation tracing example](/images/integrations/automation/automation-tracing.png)
 
 The above screenshot shows a previous run of an automation. The automation is displayed using an interactive graph, highlighting which path the automation took. Each node in the graph can be clicked to view the details on what happened with the automation during that specific step. It traces the complete run of an automation.
 
-The debugging screen is split into four features, the first being the Step Details which provides all details for each step of the automation. The second feature is the Trace Timeline which the screenshot above shows and where the automation can be followed on a timeline. The next is Related logbook entries, as the name says a logbook for all the entries related to the specific trace. The last two features are Automation Config and optionally Blueprint Config for the automation YAML code.
+The right side of the trace screen has tabs with more information:
+
+- Step Details shows data and results of the step that is currently highlighted.
+- Automation Config shows the full YAML configuration at the time the automation was run.
+- Trace Timeline, shown in the screenshot above, lists the steps that were executed and their timing.
+- Related logbook entries, as the name says a logbook for all the entries related to the specific trace.
+- Blueprint Config will only be shown if the automation was created from a {% term blueprint %}.
+
+The top bar shows the date and time the automation was triggered. Use the left and right arrows to view previous runs of the automation.
 
 Automations created in YAML must have an [`id`](/docs/automation/yaml/#migrating-your-yaml-automations-to-automationsyaml) assigned in order for debugging traces to be stored.
 
-#### Traces
+### Trace configuration
 
 The last 5 traces are recorded for all automations. It is possible to change this by adding the following code to your automation.
 
+{% raw %}
+
 ```yaml
 trace:
-    stored_traces: 1
+  stored_traces: 20
 ```
 
-[template]: /docs/configuration/templating/
-
-## Testing your automation
-
-It is generally a difficult task to test an automation, especially if it includes several triggers and some conditions.
```

Comment:
Why remove all this information about testing with triggers? I see that there is a more generic note above that triggers can't be tested that way. But for example the information that the test won't work if the automation uses data from triggers is no longer there. Was that intentional?

Reply:
I tried to rearrange to present the simplest ways of testing first, the info here ended up near the top under "running the whole automation".  I tried to phrase things more "here's how to test these things" instead of starting with "this is difficult and trigger is useless".

Reply:
I tried to rearrange to present the simplest ways of testing first, the info here ended up near the top under "running the whole automation". I tried to phrase things more "here's how to test these things" instead of starting with "this is difficult and trigger is useless".

Reply:
Yes, I saw that. And I agree with you. I also think it is better to first give instructions on what and how you can test. And then mention what is tricky. When I see that things are deleted, the first thing I do is check if information gets lost. So I'm wondering if it makes sense to add that back somewhere?

Reply:
haha, just realized you beat me to it and added a section. Thx!